Formula/pdal.rb

```diff
@@ -12,9 +11,14 @@ class Pdal < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "boost"
   depends_on "gdal"
   depends_on "laszip" => :optional
+  depends_on "postgresql" => :optional
+  depends_on "curl" => :optional
+  depends_on "python" => :optional
```

Comment:
This should simply work with the system `python`, not require Homebrew's

Reply:
We do need Numpy installed. I wasn't sure if we should check for system Python, Homebrew Python 2, or Homebrew Python 3.